docs/health/index.rst

```diff
@@ -2,62 +2,140 @@
 
 pyATS Health Check
 ==================
-Collect and monitor the state of your devices as your testscript is executing with pyATS Health Check. It can collect traceback, core files, etc. pyATS Health Check is all yaml driven and it is based on :ref:`Blitz<blitz>`. All Blitz functionalities are fully supported in health check.
+By using pyATS Health Check, it is possible to check the CPU load, memory usage, detection of specific log messages, and whether or not the core file created in the event of a crash or malfunction is generated.
 
-The health check is driven by a `health_file` which is provided at run time. There is two different mechanism to run the checks:
+pyATS Health Check currently supports the following 4 checks by default.
```

Comment:
Is it 4 or 5? health check, cpu, memory, logging and core, right?

```suggestion
pyATS Health Check currently supports the following 5 checks by default.
```

Reply:
it's 4. (cpu, memory, logging and core). so no need to change.